docs/features-roadmap.md

```diff
@@ -3,13 +3,89 @@ title: Supported features and Roadmap
 
 ## Supported Features
 
-- Search
-- MMB
+Margarita is separated to 4 main parts of the app:
 
-## Roadmap
+- [Search for connections](features-roadmap#search-for-connections)
+- [Booking](features-roadmap#booking)
+- [Manage my booking](features-roadmap#manage-my-booking)
+- [Payments](features-roadmap#payments)
+
+### Search for connections
+
+Get an access to the database of 15 billion connections virtually interlined by <a href="http://kiwi.com" target="_blank">Kiwi.com</a>.
+
+#### Search form
+
+- Initial screen of whole app. User can set here all options about connection he is looking for.
+
+<div>
+<img src="./assets/screens/search.png" alt="Search" title="Search" width="250"/>
+<img src="./assets/screens/search-passenger.png" alt="" title="" width="250"/>
+<img src="./assets/screens/search-trip-type.png" alt="" title="" width="250"/>
+</div>
+
+#### Calendar
+
+- User can choose if he wants single date, range date or specify how many night he wants to stay in selected destination.
+
+<div>
+<img src="./assets/screens/range-date-picker.png" alt="" title="" width="250"/>
+<img src="./assets/screens/range-date-picker-nights.png" alt="" title="" width="250"/>
+</div>
+
+#### Place Picker
+
+- We use Tequila for search and autocomplete places in input field. It's also possible to add multiple places for origin and destination.
+
+<div>
+<img src="./assets/screens/place-picker.png" alt="" title="" width="250"/>
+</div>
+
+#### Results
+
+Retrieved connections can be sorted according to price, time or best results. When user click item modal with more info about connection is shown.
 
-- Do more things
+<div>
+<img src="./assets/screens/results.png" alt="" title="" width="250"/>
+<img src="./assets/screens/result-detail.png" alt="" title="" width="250"/>
+<img src="./assets/screens/result-detail-expanded.png" alt="" title="" width="250"/>
+</div>
+
+### Booking
+
+Use our powerful infrastructure to book the trips and gain your commission - the best one in the industry.
+
+<div>
+<img src="./assets/screens/booking.png" alt="" title="" width="250"/>
+<img src="./assets/screens/booking-passenger.png" alt="" title="" width="250"/>
+</div>
+
+### Manage my booking
+
+Manage my booking allows users to modify the trip, edit passenger’s details or purchase additional services.
+
+<div>
+<img src="./assets/screens/mmb.png" alt="" title="" width="250"/>
+</div>
+
+#### Booking detail
+
+<div>
+<img src="./assets/screens/booking-detail.png" alt="" title="" width="250"/>
+<img src="./assets/screens/booking-detail-map.png" alt="" title="" width="250"/>
+<img src="./assets/screens/booking-detail-map2.png" alt="" title="" width="250"/>
+</div>
+
+### Payments
+
+Payments allow your users to pay instantly for their bookings, right within the app.
+
+<div>
+<img src="./assets/screens/payments.png" alt="" title="" width="250"/>
+</div>
+
+## Roadmap
```

Comment:
As briefly discussed in issue comment we can probably remove this section. I don't think there is currently any big thing left in Margarita roadmap and project is basically just being improved.
Instead of it we can maybe mention something like that we value every input and if somebody missing any feature, it can be requested like it's described in other docs section https://kiwicom.github.io/margarita/docs/questions-bugs-feature-requests .

Reply:
Should I remove Roadmap and also rename the page?

Reply:
Yes we don't have any roadmap and I don't expect that we will have

Reply:
Yes, I also think that it makes sense to remove any mention of roadmap. Probably more pages will need small update after that ( for example https://github.com/kiwicom/margarita#app-features ). So just search repo for any `roadmap` occurrence.

Reply:
Done